src/bindings/jna/libelektra4j/src/main/java/org/libelektra/Key.java

```diff
@@ -472,12 +453,11 @@ public Key currentMeta() {
 	/**
 	 * Helper function to copy some meta information from a source Key to this key.
 	 *
-	 * @param source
-	 *            Key object that is used as source
-	 * @param metaName
-	 *            Key name of the meta to be copied
-	 * @return 1 if meta was successfully copied, 0 if source doesn't contain the required meta and nothing had to be done, -1 in case of an
-	 *         error or if the source parameter was null
+	 * @param source   Key object that is used as source
```

Comment:
Did you try to build the docu? Does it look nice? https://doc.libelektra.org/api/current/html/classorg_1_1libelektra_1_1KeySet.html

Reply:
To me it looks nice: